lib/tasks/volume_tracking.rake

```diff
@@ -16,4 +16,15 @@ namespace :volume_tracking do
       end
     end
   end
+
+  task update_aliquots_with_uuid: :environment do
```

Comment:
`add_missing_uuids_to_aliquots`.  Would this be better calling your other rake task instead of adding the implementation a second time?

See https://stackoverflow.com/a/1290119/772095 showing you should just be able to add it similarly to the way `:environment` is specified here.  You can also do it explicitly with `Rake::Task['task_name'].invoke` inside the block.

Reply:
Good spot. Sorry, I think this is not required here. That was something added initially which was not removed.